modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -51,12 +51,52 @@ Before configuring host network interfaces, review the OpenShift Container Platf
 ----
 <1> Add NMState YAML syntax to configure host interfaces.
 
+During node configuration, ensure you are creating PTR records, so that the nodes are not displayed as `localhost`.
```

Comment:
I think this line should be emphasized more, as this is an essential required configuration from the client's side.
Also maybe not mention specifically that the nodes will be named localhost, just that it is essential for correct configuration of hostname.